data/json/monsters/insect_spider.json

```diff
@@ -2969,5 +2969,379 @@
+    "hp": 90,
+    "speed": 80,
+    "material": [ "iflesh" ],
+    "symbol": "w",
+    "color": "brown",
+    "armor_bash": 15,
+    "armor_cut": 18,
+    "armor_bullet": 20,
+    "armor_elec": 2,
+    "vision_day": 10,
+    "vision_night": 15,
+    "harvest": "shellfish",
+    "reproduction": { "baby_monster": "mon_woodlouse_manca", "baby_count": 5, "baby_timer": 210 },
+    "petfood": { "food": [ "CATTLEFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
+    "fear_triggers": [ "FIRE", "HURT" ],
+    "flags": [ "SEES", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SMELLS" ]
+  },
+  {
+    "id": "mon_woodlouse_manca",
+    "type": "MONSTER",
+    "name": { "str": "baby woodlouse", "str_pl": "baby woodlice" },
+    "description": "Woodlice reproduce by carrying their eggs in a pouch and releasing them as juveniles.  Their mancae look like small adults except for their pale color.",
+    "copy-from": "mon_woodlouse",
+    "proportional": { "hp": 0.25, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
+    "symbol": "w",
+    "color": "white",
+    "volume": "5 L",
+    "weight": "5 kg",
+    "upgrades": { "half_life": 14, "into": "mon_woodlouse" }
+  },
+  {
+    "id": "mon_diving_larva",
+    "type": "MONSTER",
+    "name": { "str": "giant water tiger" },
+    "description": "The predatory, aquatic larva of a diving beetle grown to large proportions.  It is quite slow but still a good ambusher.",
+    "species": [ "INSECT" ],
+    "bodytype": "insect",
+    "material": [ "iflesh" ],
+    "volume": "8 L",
+    "weight": "10 kg",
+    "hp": 35,
+    "speed": 50,
+    "symbol": "b",
+    "color": "light_gray",
+    "default_faction": "aquatic_predator",
+    "aggression": 8,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 5 } ],
+    "dodge": 2,
+    "vision_day": 10,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "fear_triggers": [ "HURT" ],
+    "flags": [ "AQUATIC", "SEES", "WATER_CAMOUFLAGE" ],
+    "upgrades": { "age_grow": 21, "into": "mon_diving_beetle_small" }
+  },
+  {
+    "id": "mon_diving_beetle_small",
+    "type": "MONSTER",
+    "name": { "str": "giant diving beetle" },
+    "description": "Water beetles can be found in all freshwater environments, with diving beetles being their predatory variant.  These however seem to have grown far larger with the Cataclysm.",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "default_faction": "beetle",
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 90,
+    "speed": 50,
+    "material": [ "iflesh" ],
+    "symbol": "B",
+    "color": "light_gray",
+    "aggression": 15,
+    "morale": 30,
+    "melee_skill": 5,
+    "melee_dice": 4,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "dodge": 1,
+    "armor_bash": 1,
```

Comment:
Why are the beetles less armored than a cicada?

Reply:
swapped the resistances, gave cicadas the same resistance as a water scorpion and the diving bettle was give half of the resistance of a woodlouse (they are quite tanky), the original reasoning was that i nerfed the diving beetle to giant spider stats because it was otherwise too busted even for aquatic predator standards.